web/locales/sv.json

```diff
@@ -48,8 +50,8 @@
         "carbonintensity": "Utsläpp av koldioxid"
     },
     "tooltips": {
-        "electricityprice": "Elpris (nästa dag)",
-        "fossilfuels": "fossila bränslen",
+        "lowcarbon": "CO2-neutralt",
```

Comment:
The translation shouldn't be "co2 neutral", it should be "low carbon" (which is not the same)

Reply:
Yes, i know. But i could not figurer out a correct translation that works in Swedish.

What is the definition of "low carbon"? How do you calculate it?
Can i use "fossil-free" instead?

Reply:
Low carbon is indeed the same as fossil-free. In Danish it's hard to find an equivalent of "low-carbon", but "fossilfri" works fine, so I like your suggestion.

Reply:
Great  :-)
I have fix that now